fe/topology.py

```diff
@@ -55,6 +61,8 @@ def standard_qlj_typer(mol):
             val = (0.0, 0.35, 1.0)
         elif a_num == 17:
             val = (0.0, 0.35, 1.0)
+        elif a_num == 35:
```

Comment:
Are there other elements `standard_qlj_typer` should cover? (I imagine this should eventually have the same elemental coverage as OpenFF 1.1. A previous commit expanded `standard_qlj_typer` to cover chlorine. Perhaps a future commit would expand to cover iodine.)